fixpatches/src/commonMain/resources/log4j.properties

```diff
@@ -0,0 +1,9 @@
+# Set root logger level to DEBUG and its only appender to A1.
```

Comment:
Don't know how a lot of tags could be more readable, that simple `key` `value` pairs. Left properties for now

Reply:
`key` `value` pairs is more readable for key-values properties.
logging config can contain a configuration with hierarchy and arrays.

Agree with you here, since current logging configuration is simple,